assets/app/views/deployments.html

```diff
@@ -10,14 +10,112 @@
           <em>{{emptyMessage}}</em>
         </div>
       </div>      
-      <div style="margin-bottom: 10px;" ng-repeat="deployment in deployments">
-        <h3>{{deployment.metadata.annotations.deploymentConfig}} <span class="small">({{deployment.metadata.name}})</span></h3>
-        <div>Created: <relative-timestamp timestamp="deployment.metadata.creationTimestamp"></relative-timestamp></div>
-        <div>Status: {{deployment.metadata.annotations.deploymentStatus}}</div>
-        <div>Version: {{deployment.metadata.annotations.deploymentVersion}}</div>
-        <div>Replicas: {{deployment.spec.replicas}}</div>
-        <pod-template ng-init="podTemplate = deployment.spec.template"></pod-template>
+      <div class="tile" ng-repeat="(deploymentConfigName, deploymentConfig) in deploymentConfigs">
+        <div> 
+          <h2>{{deploymentConfigName}}</h2>
+          <dl class="dl-horizontal left indent">
+          <div>
+              <dt>Created:</dt>
+              <dd><relative-timestamp timestamp="deploymentConfig.metadata.creationTimestamp"></relative-timestamp></dd>
+          </div>
+          <div>
+              <dt>Labels:</dt>
+              <dd>{{deploymentConfig.metadata.labels}}</dd>
+          </div>
+          <div>
+              <dt>Latest Version:</dt>
+              <dd>{{deploymentConfig.latestVersion}}</dd>
+          </div>
+          <div>
+              <dt>Strategy:</dt>
+              <dd>{{deploymentConfig.template.strategy.type}}</dd>
+          </div>
+
+          <div>
+            <h3>Triggers:</h3>
+            <dl class="dl-horizontal left indent">
+              <div ng-repeat="trigger in deploymentConfig.triggers">
+                <span ng-switch="trigger.type">
+                  <span ng-switch-when="ImageChange">
+                    TODO: ImageChange                    
+                  </span>
+                  <span ng-switch-when="ConfigChange">                    
+                    TODO: ConfigChange                    
+                  </span>
+                  <span ng-switch-default>{{trigger.type}}</span>
+                </span>
+              </div>
+            </dl>
+
+            <dl class="dl-horizontal left indent">
```

Comment:
move the manual ones first, since they're the ones you can actually act on. If we have a button that works in the UI, I question showing the CLI command